README.md

```diff
@@ -106,6 +108,19 @@ complete -o default -F __start_corectl <myalias>
 ```
 where `<myalias>` should be substituted for whatever you wish to call `corectl`.
 
+### Windows
```

Comment:
Adding bash-completion to git bash is still applicable for Windows users as well.

Reply:
True!